interfaces/seccomp/template.go

```diff
@@ -549,3 +549,23 @@ pwritev
 # of socket(), bind(), connect(), etc individually.
 socketcall
 `)
+
+// bindSyscallWorkaround specifies the bind() syscall for systems which
+// don't use AppArmor but only seccomp. In those cases the net package
```

Comment:
Sorry for being perhaps pedantic, but still doesn't sound quite right. What Go uses in the net package doesn't depend on whether apparmor is enabled or not. The behavior of Go is always the same.

Please let me give it a try:
```
// Go's net package attempts to bind early to check whether IPv6 is available or not.
// For systems with apparmor enabled, this will be mediated and cause an error to be
// returned. Without apparmor, the call goes through to seccomp and the process is
// killed instead of just getting the error.
//
// For that reason once apparmor is disabled the seccomp profile is given access
// to bind, so that these processes are not improperly killed. There is on going
// work to make seccomp return an error in those cases as well and log the error.
// Once that's in place we can drop this hack.
```

How does that sound?  @jdstrand, is this right?

This explanation actually makes me wonder about the point from @mvo5 a bit more. I wonder if we shouldn't simply disable seccomp altogether if apparmor is not around, until this is fixed. We're special casing one specific call because it hurts us, but my guess is that there will be other snaps with the exact same kind of problem on different syscalls out there.